CHANGES.md

```diff
@@ -17,6 +17,7 @@
   indented less (#3964)
 - Multiline list and dict unpacking as the sole argument to a function is now also
   indented less (#3992)
+- Preserve printable quote types for debug f-strings (#4005)
```

Comment:
Not sure why you call this "printable" quotes, aren't they just string quotes?

Reply:
Only quotes that are visible in the the final string's debug replacement fields are of interest to us, as those affect the AST. I agree that 'printable' probably isn't the best name here. Maybe "In f-string debug replacement fields preserve quote types that are visible in the final string"?